internal/resolution/resolver_test.go

```diff
@@ -19,112 +18,113 @@ import (
 )
 
 func TestOperatorResolver(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Operator Resolver Suite")
-}
-
-func FakeClient(objects ...client.Object) client.Client {
-	scheme := runtime.NewScheme()
-	if err := v1alpha1.AddToScheme(scheme); err != nil {
-		panic(fmt.Sprintf("error creating fake client: %s", err))
-	}
-	return fake.NewClientBuilder().WithScheme(scheme).WithObjects(objects...).Build()
-}
 
-var testEntityCache = map[deppy.Identifier]input.Entity{
-	"operatorhub/prometheus/0.37.0": *input.NewEntity("operatorhub/prometheus/0.37.0", map[string]string{
+	testEntityCache := map[deppy.Identifier]input.Entity{"operatorhub/prometheus/0.37.0": *input.NewEntity("operatorhub/prometheus/0.37.0", map[string]string{
 		"olm.bundle.path": `"quay.io/operatorhubio/prometheus@sha256:3e281e587de3d03011440685fc4fb782672beab044c1ebadc42788ce05a21c35"`,
 		"olm.channel":     "{\"channelName\":\"beta\",\"priority\":0,\"replaces\":\"prometheusoperator.0.32.0\"}",
 		"olm.gvk":         "[{\"group\":\"monitoring.coreos.com\",\"kind\":\"Alertmanager\",\"version\":\"v1\"}, {\"group\":\"monitoring.coreos.com\",\"kind\":\"Prometheus\",\"version\":\"v1\"}]",
 		"olm.package":     "{\"packageName\":\"prometheus\",\"version\":\"0.37.0\"}",
 	}),
-	"operatorhub/prometheus/0.47.0": *input.NewEntity("operatorhub/prometheus/0.47.0", map[string]string{
-		"olm.bundle.path": `"quay.io/operatorhubio/prometheus@sha256:5b04c49d8d3eff6a338b56ec90bdf491d501fe301c9cdfb740e5bff6769a21ed"`,
-		"olm.channel":     "{\"channelName\":\"beta\",\"priority\":0,\"replaces\":\"prometheusoperator.0.37.0\"}",
-		"olm.gvk":         "[{\"group\":\"monitoring.coreos.com\",\"kind\":\"Alertmanager\",\"version\":\"v1\"}, {\"group\":\"monitoring.coreos.com\",\"kind\":\"Prometheus\",\"version\":\"v1alpha1\"}]",
-		"olm.package":     "{\"packageName\":\"prometheus\",\"version\":\"0.47.0\"}",
-	}),
-	"operatorhub/packageA/2.0.0": *input.NewEntity("operatorhub/packageA/2.0.0", map[string]string{
-		"olm.bundle.path": `"foo.io/packageA/packageA:v2.0.0"`,
-		"olm.channel":     "{\"channelName\":\"stable\",\"priority\":0}",
-		"olm.gvk":         "[{\"group\":\"foo.io\",\"kind\":\"Foo\",\"version\":\"v1\"}]",
-		"olm.package":     "{\"packageName\":\"packageA\",\"version\":\"2.0.0\"}",
-	}),
-}
+		"operatorhub/prometheus/0.47.0": *input.NewEntity("operatorhub/prometheus/0.47.0", map[string]string{
```

Comment:
nit: does this need the additional space?

Reply:
go fmt creates this additional spacing.